src/lib.rs

```diff
@@ -49,6 +52,21 @@ impl FileError {
                     ret.push_str(&epath.display().to_string());
                     ret.push_str(" does not exist!");
                 }
+                Anchor(url, anchor) => {
+                    ret.push_str("\n\tAnchor #");
+                    ret.push_str(anchor);
+                    ret.push_str(" in linked file at URL ");
+                    ret.push_str(url.as_str());
+                    ret.push_str(" does not exist!");
```

Comment:
This is not worth the tiny performance increase I think.
```suggestion
                    ret.push_str(&format!("\n\tAnchor #{} in linked file at URL {} does not exist!", anchor, url);
```
Same for `AnchorRange` below.